src/onegov/org/forms/meeting.py

```diff
@@ -1,24 +1,102 @@
 from __future__ import annotations
 
-from wtforms import DateTimeField
+from wtforms.validators import Optional, InputRequired
+from wtforms import StringField
 
 from onegov.form import Form
+from onegov.form.fields import TimezoneDateTimeField, ChosenSelectMultipleField
 from onegov.org.forms.fields import HtmlField
 from onegov.org import _
-from wtforms.validators import Optional
+
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from onegov.org.models import Meeting
 
 
 class MeetingForm(Form):
 
-    start_datetime = DateTimeField(
+    title = StringField(
+        label=_('Title'),
+        validators=[InputRequired()],
+    )
+
+    start_datetime = TimezoneDateTimeField(
+        timezone='Europe/Zurich',
         label=_('Start'),
         validators=[Optional()],
     )
 
+    end_datetime = TimezoneDateTimeField(
+        timezone='Europe/Zurich',
+        label=_('End'),
+        validators=[Optional()],
+    )
+
     address = HtmlField(
-        label=_('Portrait'),
+        label=_('Address'),
+        validators=[InputRequired()],
+        render_kw={'rows': 3}
     )
 
     description = HtmlField(
         label=_('Description'),
+        validators=[Optional()],
+        render_kw={'rows': 5}
+    )
+
+    meeting_items = ChosenSelectMultipleField(
+        label=_('Agenda Items'),
+        choices=[],
+        validators=[Optional()]
     )
+
+    def on_request(self) -> None:
+        from onegov.org.models import MeetingItem
+
+        meetings = (
+            self.request.session.query(MeetingItem)
+            .order_by(MeetingItem.number, MeetingItem.title)
+            .all()
+        )
+        self.meeting_items.choices = [
+            (item.id.hex, item.display_name)
+            for item in meetings
+        ]
+
+    def populate_obj(self, obj: object, *args: Any, **kwargs: Any) -> None:
+        from onegov.org.models import MeetingItem
+
+        super().populate_obj(obj, exclude={'meeting_items'})
+
+        meeting: Meeting = obj  # type:ignore[assignment]
```

Comment:
```suggestion
    def populate_obj(  # type: ignore[override]
        self,
        obj: Meeting,  # type: ignore[override]
        exclude: Collection[str] | None = None,
        include: Collection[str] | None = None
    ) -> None:
        from onegov.org.models import MeetingItem

        super().populate_obj(obj, exclude={'meeting_items', *(exclude or ())})

        meeting = obj
```
It's slightly more robust to add the model to signature and use ignore the override errors, since you will still get an error if you pass in something other than a `Meeting` into that function this way.